compiler/luci/pass/src/ResolveCustomOpMaxPoolWithArgmaxPass.test.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/ResolveCustomOpMaxPoolWithArgmaxPass.h"
+
+#include <gtest/gtest.h>
+
+TEST(ResolveCustomOpMaxPoolWithArgmaxPassTest, name)
+{
+  luci::ResolveCustomOpMaxPoolWithArgmaxPass pass;
+  auto const name = pass.name();
+  ASSERT_NE(nullptr, name);
+}
```

Comment:
(optional) Testing with real graph would be good like `ResolveCustomOpBatchMatMulPass.test`.

Reply:
But seems bothering you to check above `after` graph..

Reply:
(optional) And, it would be good if current pass passes `circle2circle-dredd-recipe-test`. Value tests are more correct than code review with eyes..

Reply:
@mhs4670go 
I plan to add dredd tests to check pass works, but not value test.

It is hard to perform automated value test for now, because dredd tests get reference data using tflite model, but MaxPoolWithArgmax is a custom operation and available only in "big" TensorFlow.

To perform value tests I see two ways:
- Use [select operators](https://www.tensorflow.org/lite/guide/ops_select) in testing system
- Introduce new test suite that will compare circle execution with original tensor flow model (AFAIK there are no such test suite for now)

Reply:
@binarman Ah, you're right. I misunderstood for a moment. Thank you for detail:)